### PR TITLE
Add Python 3.11

### PR DESCRIPTION
**Pull request by abdulrabbani00**

### CODE REVIEW

File: src/postgres/devcontainer-template.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
             "description": "Python version (use -bullseye variants on local arm64/Apple Silicon):",
             "proposals": [
                 "3",
```
Comment (samruddhikhandale) [inline src/postgres/devcontainer-template.json:14]: Can we bump the patch version here? 👇 
https://github.com/devcontainers/templates/blob/8bcdbe508dc96ed6feb427aad35ed64563085085/src/postgres/devcontainer-template.json#L3

The Template will only be published if the version does not exist, hence needs a version bump.

File: src/python/devcontainer-template.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
             "description": "Python version (use -bullseye variants on local arm64/Apple Silicon):",
             "proposals": [
```
Comment (samruddhikhandale) [inline src/python/devcontainer-template.json:13]: Same here, can we bump the version for this template as well?

File: src/postgres/devcontainer-template.json [hunk 3]
```diff
@@ -12,6 +12,7 @@
...
+                "3.11",
```
Comment (samruddhikhandale) [inline src/postgres/devcontainer-template.json:15]: Can we also add `3.11-bullseye` & `3.11-buster` to the proposals?

File: src/postgres/devcontainer-template.json [hunk 4]
```diff
@@ -12,6 +12,7 @@
...
                 "3.10",
                 "3.9",
                 "3.8",
```
Comment (samruddhikhandale) [inline src/postgres/devcontainer-template.json:18]: Can we change the `default` to `3.11`?

File: src/python/devcontainer-template.json [hunk 5]
```diff
@@ -12,6 +12,7 @@
...
                 "3",
+                "3.11",
```
Comment (samruddhikhandale) [inline src/python/devcontainer-template.json:15]: Can we also add `3.11-bullseye` & `3.11-buster` to the proposals?

File: src/python/devcontainer-template.json [hunk 6]
```diff
@@ -12,6 +12,7 @@
...
                 "3.10",
                 "3.9",
```
Comment (samruddhikhandale) [inline src/python/devcontainer-template.json:17]: Can we change the `default` to `3.11`?

**Review by samruddhikhandale — COMMENTED** [review]
Thanks for opening the PR. Left some comments!

File: src/python/devcontainer-template.json (hunk 6 above)
Comment (abdulrabbani00) [reply src/python/devcontainer-template.json:17]: hahahah lol

**Review by samruddhikhandale — APPROVED** [review]
Looks great! Thank you 🚀

**Pull request merged**